.releaserc.json

```diff
@@ -0,0 +1,9 @@
+{
+  "plugins": [
+    "@semantic-release/commit-analyzer",
+    "@semantic-release/release-notes-generator",
```

Comment:
* According to the plugin descriptions [here](https://semantic-release.gitbook.io/semantic-release/extending/plugins-list#official-plugins), these two plugins are already included by default when semantic-release is ran? 🤔 
* Seems like we'd also want to include `@semantic-release/changelog`.

Reply:
Ah cool I hadn't seen that plugins page. I basically copied this configuration from the [Usage directions](https://github.com/semantic-release/git#usage) of the `@semantic-release/git` plugin. Looks like we could also make this [shareable](https://github.com/semantic-release/semantic-release/blob/master/docs/usage/shareable-configurations.md) once we confirm this works.

Good call on the `@semantic-release/changelog` -- I'll read up on how to configure.